CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,75 @@
+# Contributor Covenant Code of Conduct
+
+# **Our Pledge**
```

Comment:
[nit] Bold (**) won't have any effect on titles right?

Reply:
Yeah you're right! They have no effect. Removed them. 👌